sys/include/net/gnrc/ipv6/nib/conf.h

```diff
@@ -184,6 +184,16 @@ extern "C" {
 #define GNRC_IPV6_NIB_NUMOF                 (4)
 #endif
 
+/**
+ * @brief   Number of off-link entries in NIB
+ *
+ * @attention   This number has direct influence on the maximum number of
```

Comment:
just to clarify: how exactly does `GNRC_IPV6_NIB_OFFL_NUMOF` influences the max. number? I would assume it **is** the max. number, or is there another number that is added to that?